src/test/java/ch/sportchef/business/authentication/control/AuthenticationServiceTest.java

```diff
@@ -76,6 +73,22 @@
     @Inject
     private ConfigurationService configurationServiceMock;
 
+    private SmtpServer smtpServer;
+
+    @Before
+    public void setup() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [You may have misspelled a JUnit framework method (setUp or tearDown)](https://www.codacy.com/app/McPringle/sportchef/file/2485993635/issues/source?bid=3149602&fileBranchId=3265763#l79)